google/cloud/forseti/common/gcp_api/securitycenter.py

```diff
@@ -130,14 +130,14 @@ def create_finding(self, finding, source_id=None, finding_id=None):
             dict: An API response containing one page of results.
         """
         try:
-            LOGGER.debug('Creating finding with beta api.')
+            LOGGER.debug('Creating finding with GA API.')
```

Comment:
At this point, we should not explicitly mention `GA API` anymoe, since GA and API are the only options.
```
LOGGER.debug('Creating finding.')
```
Please remove everywhere.